c/datatablemodule.cc

```diff
@@ -199,6 +200,7 @@ void DatatableModule::init_methods() {
   add(METHOD0(is_debug_mode));
   add(METHOD0(has_omp_support));
   add(METHODv(aggregate));
+  add(METHODv(ftrl));
```

Comment:
Note that macros such as `METHOD0`, `METHODv` etc will soon be removed due to issue #1369. The newer mechanism for exposing API into python can be seen for example in `py_str.cc`.